### PR TITLE
Sierra journal example

### PR DESCRIPTION
**Pull request by ChristopherHilton**
### What is this PR trying to achieve?
Sample of a Sierra journal description in JSON
### Who is this change for?
Silver, then Platform team

### CODE REVIEW

File: ontologies/Journal_with_items.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+				"label": "Related to: New York lancet (New York, N.Y.)"
+			},
+			{
+				"label": "Lancet (North American ed.)"
+			},
+			{
+				"label": "Lancet (Deutsche Ausg.)"
+			}
+		],
+	"creators":
```
Comment (pollecuttn) [inline ontologies/Journal_with_items.json]: These should be more specific than creator. They are 'Authors, etc' according to the label and 'Editors' according to the 700 field

File: ontologies/Journal_with_items.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+			},
+			{
+				"label": "Wakley, Thomas, 1795-1862"
+			},
+			{
+				"label": "Sprigge, S. Squire (Samuel Squire)"
+			}
+		],
+	
+  "items":
```
Comment (pollecuttn) [inline ontologies/Journal_with_items.json]: Journals will also need holdings as well as items.

File: ontologies/Journal_with_items.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Comment line 13-22: 
+a) are we happy dealing with all these different sorts of title as subsets of alternativeTitlt?
+b) if so, keyTitle and uniformTitle need to be added to the ontology and described
+
+Comment line 43: "designation" needs to be added the ontology
+It corresponds in part to the start date and end date properties that are already there, covering dates of publication and sequential designation, but has other things mixed in like volumes
+
+Comment line 44 onwards: ISSN is not in the ontology yet
+It occupies the same field as ISBN and the distinguishing feature is apparently the format: ISSN has four digits, a slash then another four digits.
```
Comment (pollecuttn) [inline ontologies/Journal_with_items.md]: ISSN/ISBN luckily do not occupy the same field. 020 is for ISBN. 022 is for ISSN.  http://www.loc.gov/marc/bibliographic/bd01x09x.html

File: ontologies/Journal_with_items.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Comment line 58: frequency can be either current or past: the distinction lies in which MARC field it comes from (310 or 321 respectively)
+
+Comment line 88: numbering is not yet in ontology
+
+Comment line 89: supplement is not yet in ontology
+
+Comment lines 90-108: relatedMaterialNote brings together material fromn MARC 530 - the alternative formats field - and MARC 775, related to.  We would want to distinguish these?
+
+Comment, lines 111-124: MARC field 700 is a field for names addedd to a catalogue record
```
Comment (pollecuttn) [inline ontologies/Journal_with_items.md]: 700 is for personal names (i.e. names of people, not names of meetings or organisations).

File: ontologies/Journal_with_items.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Comment line 57: Format in this instance just duplicates the content of WorkType - surely we don't need both
+
+Comment line 58: frequency can be either current or past: the distinction lies in which MARC field it comes from (310 or 321 respectively)
+
+Comment line 88: numbering is not yet in ontology
+
+Comment line 89: supplement is not yet in ontology
+
+Comment lines 90-108: relatedMaterialNote brings together material fromn MARC 530 - the alternative formats field - and MARC 775, related to.  We would want to distinguish these?
```
Comment (pollecuttn) [inline ontologies/Journal_with_items.md]: Should that be 776 (Additional physical form entry) rather than 775 (other edition entry)?

**Comment by ChristopherHilton** [comment]
Merging this in the light of earlier discussion

**Pull request merged**